nss-day-cohort-48/Rare-API-Spicy-Sauce-Scouts
# Creating Comments

#### Changes Made
1. Added file `Comment` to `views` directory.
1. Modified file `comment` to include `Create ,Update ,Retrieve ,Delete ,List` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cwj-creating-comments
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
